pynio/rest.py

```diff
@@ -17,9 +17,15 @@ def __init__(self, host='127.0.0.1', port=8181, creds=None):
         self._creds = creds or ('User', 'User')
         self._url = 'http://{}:{}/{}'.format(host, port, '{}')
 
-    def _get(self, endpoint, timeout=None, data=None, retry=0):
-        '''Performs a get with some amounts of retrys'''
-        if data is not None:
+    def _get(self, endpoint, timeout=None, data=None, retry=0, json=True):
```

Comment:
Is there any chance of the scope being confused since this module imports json and it's being used as a variable name? This method uses both 'json.dumps' and 'if json' but they are both referring to a different thing. That is a little unclear.